MarkdownKit/Sources/Common/MarkdownParser.swift

```diff
@@ -159,8 +159,8 @@ open class MarkdownParser {
       (.bold, bold),
       (.italic, italic),
       (.strikethrough, strikethrough),
-      (.link, link),
```

Comment:
When the link was being parsed before, it would join the text coming after the link, and the autolink would incorrectly make the following text a part of the link.